features/step_definitions/session-storage.js

```diff
@@ -0,0 +1,14 @@
+let session;
+
+function create(_session) {
```

Comment:
I thinks the name of this function should be `store` as it is not creating anything. What if instead creating a session storage we create a session provider. Which will create the session internally. It can export three things:
- start (asynchronous function)
- stop (asynchronous function)
- session (synchronous function)
What do you think?